projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -70,17 +84,45 @@ export class NavigationListComponent implements OnChanges {
 
   public activeItem$?: Observable<NavItemLinkConfig | undefined>;
 
+  public navItems$?: Observable<NavItemConfig[]>;
+
   public constructor(
     private readonly navigationService: NavigationService,
     private readonly activatedRoute: ActivatedRoute,
-    private readonly navListComponentService: NavigationListComponentService
+    private readonly navListComponentService: NavigationListComponentService,
+    private readonly timeRangeService: TimeRangeService,
+    private readonly featureStateResolver: FeatureStateResolver
   ) {}
 
   public ngOnChanges(): void {
     this.navItems = this.navListComponentService.resolveFeaturesAndUpdateVisibilityForNavItems(this.navItems);
-    this.activeItem$ = this.navigationService.navigation$.pipe(
-      startWith(this.navigationService.getCurrentActivatedRoute()),
-      map(() => this.findActiveItem(this.navItems))
+
+    this.navItems$ = this.featureStateResolver
+      .getCombinedFeatureState([ApplicationFeature.PageTimeRange, ApplicationFeature.NavigationRedesign])
```

Comment:
why should it depend on PageTimeRange? Shouldn't it just depend on `NavigationRedesign`?

Reply:
@Christian862 @aaron-steinfeld  If we disable PageTimeRange and enable NavigationRedesign, then I suppose the UI will show TR anywhere since the top header will be hidden.

I think we should add a FF dependency for `NavigationRedesign` on `PageTimeRange`. NavigationRedesign can only be enabled if  PageTimeRange is enabled.

Reply:
I'm fine sequencing them if there is any real complexity to handle all four combinations.